### ISSUE TITLE
Add pileup interestingness score

### ISSUE THREAD

**Issue by hammer** [hammerlab/cycledash#86]
When we call a lot of variants it would be nice to give some guidance to the user as to what variants should be inspected

**Comment by hammer** [comment]
Some random thoughts: coverage depth, allele frequency, functional significance of the location, population frequency of the variant

**Comment by ihodes** [comment]
Could also be fun to extract "interesting" features from a list of variants we already deem interesting (e.g. a list of a bunch of false negatives), then use those features for this issue.

**Comment by danvk** [comment]
To make sure we're on the same page:

- Higher coverage depth = more interesting
- Higher allele frequency = more interesting
- In a gene = more interesting
- Higher population frequency = more interesting? less interesting?

**Comment by hammer** [comment]
Yeah I guess the hard part is how to validate the score. When I say "interesting", I guess I mean the variant (or genotype) is worth close inspection. We could use some user feedback mechanism to help us grade whether we're directing attention effectively.

**Comment by ihodes** [comment]
https://peerj.com/articles/796/
